backend/census_historical_migration/transforms/xform_uppercase_y_or_n.py

```diff
@@ -0,0 +1,25 @@
+from ..exception_utils import DataMigrationValueError
+
+from django.conf import settings
+
+
+def uppercase_y_or_n(value):

```

Comment:
`string_to_bool` will need a similar change to account for `y/n`

Reply:
Isn't it being converted to Y/N before it gets to `string_to_bool`, though?

Reply:
I thought that too, but it already performs an `upper()` before in that method